### ISSUE TITLE
`createStaticPaths` callback receives incorrect `langCode`

### ISSUE THREAD

**Issue by Klavionik** [Alexandre-Fernandez/astro-i18n#42]
**Describe the bug**
I'm building a site with Astro using content collections, and I'm facing something that might be a bug in `createStaticPaths` (or just me doing something wrong).

It all works well in development, but when building a site, the `createStaticPaths` callback receives an incorrect `langCode` (always the default one). The reason seems to be that `import.meta.url` is different when building: there is no language code in the file import url. 

Because of this, content entries don't get translated, they all are in the default language.

I added a few console.logs inside `getStaticPaths`, ran `npm run build` and this is what I see:

![Screenshot from 2023-07-23 22-40-59](https://github.com/Alexandre-Fernandez/astro-i18n/assets/56698047/4b88a9f1-4d4c-4ca4-aca8-2d1ca175ec44)

Despite building a page from `src/pages/en/...` directory, the langCode is `ru`. And `import.meta.url` seems to be inside of the `dist` directory.

Right now I'm able to work around this if I change this code in the autogenerated `src/pages/en/...` directory:
```diff
export const getStaticPaths = (props) => proxyGetStaticPaths({
	...props,
-	langCode: extractRouteLangCode(import.meta.url),
+	langCode: "en",
})
```

This way I just hardcode the `en` locale for content pages inside the `en` directory. Then everything works fine.

**To Reproduce**
I put up a live demo of the bug, based on an Astro Blog example: https://effortless-malabi-bc87b1.netlify.app/
Navigate to Blog, then change the language to EN. Verify that the only entry's title says "First post (EN)" and open it. You will see the post, but now its title will be "First Post (RU)".

**Expected behavior**
I expect `langCode` to be correct when building content collection pages.

**Mandatory reproduction repository**
https://github.com/Klavionik/astro-i18n-bug

**Comment by Alexandre-Fernandez** [comment]
This and many other bugs will be fixed with 2.0 which I'm coding right now, if you want to fix it you can take a look at the code here https://github.com/Alexandre-Fernandez/astro-i18n/blob/main/src/core/fs/generators/page.proxy.ts .

**Comment by Alexandre-Fernandez** [comment]
This should work in the new version, however you get the langcode in the `astroI18n` singleton instead of in the params.
